README.md

```diff
@@ -12,121 +12,7 @@ Exoframe is a self-hosted tool that allows simple one-command deployments using
-
-Exoframe also supports running in [Swarm mode](https://docs.docker.com/engine/swarm/).  
-To run Exoframe server in swarm, you need to do the following:
-
-1.  Make sure you have Docker on your host.
-2.  Make sure your Docker has [Swarm mode enabled](https://docs.docker.com/engine/swarm/swarm-tutorial/create-swarm/).
-3.  Pull and run Exoframe server using Docker on your manager node:
-
-```
-docker service create \
-  --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
-  --mount type=bind,source=/path/to/exoframe-folder,target=/root/.exoframe \
-  --mount type=bind,source=/home/user/.ssh/authorized_keys,target=/root/.ssh/authorized_keys,readonly \
-  -e EXO_PRIVATE_KEY=your_private_key \
-  --label traefik.backend=exoframe-server \
-  --label traefik.frontend.rule=Host:exoframe.your-host.com \
-  --label traefik.port=8080 \
-  --constraint=node.role==manager \
-  --name exoframe-server \
-  exoframe/server
-```
-
-Note that both Exoframe server and Traefik will be run on your manager node.
-
-## Configuration
-
-Exoframe stores its config in `~/.exoframe/server.config.yml`.  
-Currently it contains the following settings:
-
-```yaml
-# whether debug mode is enabled, default "false"
-debug: false
-
-# whether to enable letsencrypt, default "false"
-letsencrypt: false
-
-# email used for letsencrypt
-letsencryptEmail: your@email.com
-
-# whether to apply gzip compression, default "true"
-compress: true
-
-# base top-level domain to use for deployments without domains specified, default "false"
-# used as postfix, e.g. if you specify ".example.com" (dot is auto-prepended if not present)
-# all your deployments will be autodeployed as "deployment-id.example.com"
-baseDomain: false
-
-# CORS support; can be "true" ("*" header) or object with "origin" property, default "false"
-cors: false
-
-# Traefik image to be used; set to "false" to disable traefik management, default "traefik:latest"
-traefikImage: 'traefik:latest'
-
-# Traefik container name, default "exoframe-traefik"
-traefikName: 'exoframe-traefik'
-
-# Additional Traefik start args, default []
-traefikArgs: []
-
-# Network used by traefik to connect services to, default "exoframe"
-exoframeNetwork: 'exoframe'
-
-# server image update channel; can be "stable" or "nightly", default "stable"
-updateChannel: 'stable'
-
-# path to folder with authorized_keys, default "~/.ssh"
-publicKeysPath: '/path/to/your/public/keys'
-
-# whether Exoframe server whould be running in swarm mode, default "false"
-swarm: false
-```
-
-_Warning:_ Most changes to config are applied immediately. With exception of Letsencrypt config. If you are enabling letsencrypt after Traefik instance has been started, you'll need to remove Traefik and then restart Exoframe server for changes to take effect.
+For more details on how to get up an runing please falow the link [ How to setup exoframe-server](https://github.com/exoframejs/exoframe/tree/master/docs).
```

Comment:
Please, proof read that line a bit, there's a couple of typos here as well as some unneeded capitalization and extra spaces :)